### ISSUE TITLE
consistency in case / naming

### ISSUE THREAD

**Issue by brianaAtWork** [FihoFi/ntm_common_xml#1]
The NoticeToMarinerList is of type NoticeToMarinerList
Seems the pattern should be maintained for Notices => Notices and Notice => Notice (instead of NoticeToMariner or notice should be NoticeToMariner of type NoticeToMariner)

**Comment by brianaAtWork** [comment]
Also seems redundant and outside of the scope of reuse naming 8 variables within NoticeToMariner with the start of the name as "notice"....we already know it is contained within the scope of notice, so why say it again? Can these not be better harmonized with S-100 already by reusing concepts that might already exist?

items in question:
noticeIdentifier => id
noticeDatePublished => datePublished
noticeStatus => npubStatus (this would make it reuseable for all NIPWG status values)
noticeOriginal => isOriginal (this would help to signify a boolean type)
noticeOriginalSource => source (this should just be source ...if boolean above is false then what?)
noticeValidity => npubValidityText (this would help it be generic as well as tell datatype to avoid confusion)
noticeRefersToNotice => referencesNotice 
noticeCancelsNotice => cancelsNotice

**Comment by FihoFi** [comment]
I agree the consictency of naming could be improved (first comment) and further discussed (second comment).

**Comment by brianaAtWork** [comment]
> I agree the consictency of naming could be improved (first comment) and further discussed (second comment).

It would be really nice to get this tidied up first (sooner than later) so updates to the associated affected files would be less of an impact.

Can we get this process going...I will be happy to do so if we could agree on the best approach. Thanks
